examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -136,19 +146,42 @@
       }
     },
     {
+      "kind": "ServiceAccount",
+        "apiVersion": "v1",
+        "metadata": {
+            "name": "${JENKINS_SERVICE_NAME}"
+        }
+    },
+    {
+      "kind": "RoleBinding",
+      "apiVersion": "v1",
+      "metadata": {
+          "name": "${JENKINS_SERVICE_NAME}_edit"
```

Comment:
I thought we weren't going to be creating rolebindings in our templates? But I suppose in this case the experience would be even worse.